src/Layout.js

```diff
@@ -1122,8 +1122,50 @@ var computeLayout = (function() {
       child.nextAbsoluteChild = null;
     }
   }
+  
+  function layoutNode(node, parentMaxWidth, parentDirection) {
+    if (node.isDirty ||
+        !node.lastLayout || 
+         node.lastLayout.requestedHeight !== node.layout.height ||
+         node.lastLayout.requestedWidth !== node.layout.width ||
+         node.lastLayout.parentMaxWidth !== parentMaxWidth) {
+           
+      if (!node.lastLayout) {
+        node.lastLayout = {};
+      }
+      
+      node.lastLayout.requestedWidth = node.layout.width;
+      node.lastLayout.requestedHeight = node.layout.height;
+      node.lastLayout.parentMaxWidth = parentMaxWidth;
+      
+      // Reset child layouts
+      node.children.forEach(function(child) {
+        child.layout.width = undefined;
+        child.layout.height = undefined;
+        child.layout.top = 0;
+        child.layout.left = 0;
+        child.layout.bottom = 0;
```

Comment:
hmm, what's going on with those two? We shouldn't compute those. They can be obtained from width and height